### ISSUE TITLE
"Could not decode the documentation!" in Getting Started

### ISSUE THREAD

**Issue by somenxavier** [mint-lang/mint#341]
I get nothing when I run `mint docs` in the example of [Getting Started](https://www.mint-lang.com/guide/getting-started/using-the-cli)

```
$ mint init test
Mint - Initializing a new project
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
⚙ Creating directory: test
⚙ Writing initial files:
  ➔ assets/head.html
  ➔ source/Main.mint
  ➔ tests/Main.mint
  ➔ mint.json
  ➔ .gitignore
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
All done in 4.065ms!
[xan@coronari proves]$ cd test/
[xan@coronari test]$ mint install
Mint - Installing dependencies
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
There are no dependencies!

There is nothing to do!
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
All done in 110μs!
[xan@coronari test]$ mint build
Mint - Building for production
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
⚙ Ensuring dependencies... 3μs
⚙ Clearing the "dist" directory... 5.57ms
⚙ Compiling your application:
  ➔ Parsing 1 source files... 3.118ms
  ➔ Type checking: 10.79ms
  ➔ Compiling: 1.23ms
⚙ Writing index.html... 311μs
⚙ Writing manifest.json...49μs
⚙ Generating icons... 120.588ms
⚙ Creating service worker...1.479ms
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
All done in 1.797s!
[xan@coronari test]$ mint docs
Mint - Running the documentation server
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
⚙ Documentation server started on http://127.0.0.1:3002/
  Also available here http://127.0.0.1:3000/


```

Perhaps it's a documentation bug or what?

**Comment by gdotdesign** [comment]
Yes, it seems we forgot to update the documentation viewer https://github.com/mint-lang/mint-docs-viewer for the new release.

I'll tag it to be fixed in the new version.

**Comment by gdotdesign** [comment]
Fixed in 6ff4025efc66596bff2eb9fe78906805b174bb42 and https://github.com/mint-lang/mint-docs-viewer/commit/3254de87049017f85a7c8fb7110ca2b7d50d934d
